src/core/lv_obj.c

```diff
@@ -39,6 +39,14 @@
  *      TYPEDEFS
  **********************/
 
+typedef struct {
+    lv_screen_load_anim_t anim_type;
+    uint32_t duration;
+    uint32_t delay;
+    lv_obj_t * screen;
+    lv_screen_create_cb_t create_cb;
```

Comment:
Not sure if this is a good idea. Maybe don't do it. Just wanted to mention it anyways.

```suggestion
    union {
        lv_obj_t * screen;
        lv_screen_create_cb_t create_cb;
    };
```